app/components/doi-metrics-badge.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+
+
+export default Component.extend({
+  tagName: 'div',
```

Comment:
Why are you writing this, it is the default behaviour.

Reply:
doi-metrics-badge.js  has been removed